packages/providers/openai/src/azure.ts

```diff
@@ -1,6 +1,8 @@
-import { getEnv } from "@llamaindex/env";
+import { getEnv, process } from "@llamaindex/env";
+import type { FinalRequestOptions } from "openai/core.mjs";
+import pkg from "../package.json";
```

Comment:
import json directly is not recommend

Reply:
Gotcha! What do you recommend for reading the `package.json` version (syncronously) because `fs.readFile()` is not an option here. I am overriding a protected method from the base class, and I cannot make it async: 

https://github.com/run-llama/LlamaIndexTS/pull/1367/files#diff-8aef260b9972bf0ea09cdd0d888b2dc475ac64cd645cf970a26bb9e8576add97R147-R152

Reply:
OK, just double checked, it's okay here since bundler will inline the JSON to dist